tox.ini

```diff
@@ -138,7 +138,7 @@ commands = {envpython} -m pytest -vv -m "not hypothesis_slow" --cov=cloudinit --
 deps =
     {[testenv]deps}
     pytest-xdist
-commands = {envpython} -m pytest -n auto -m "not hypothesis_slow" -m "not serial" {posargs:tests/unittests}
+commands = {envpython} -m pytest -n auto -m "not hypothesis_slow" -m "not serial" --cov=cloudinit {posargs:tests/unittests}
```

Comment:
I think the vast majority of times people would want to run tests they won't be looking at a coverage report. Is there a reason we're wanting to generate it every time? Can we just run `tox -e py3-fast -- --cov=cloudinit` when we need it?

Reply:
I strongly disagree. I feel like we should be pushing for people to be more aware of coverage whenever possible. By making it the default, the developer doesn't need to go out of there way to observe the coverage. Whereas if it was not the default, I think 99% of the time people wouldn't use it, let alone even know it exists (I'm speaking more to contributors here, not maintainers, but I still think it holds somewhat true). 

Is there a disadvantage to always gathering coverage that I'm not seeing?

Reply:
Just that it's several seconds slower (this is the FAST job) and people tend to iterate with unit tests a lot more often than coverage reports. I don't really expect a random contributor to look at a coverage report. To have any meaning, they'd first need to run it against main and see how their branch compares. I doubt anybody not part of the project is doing that tbh. I myself will only look at the html ones.

This isn't a blocking comment for me though, and I don't run the tox jobs locally. We can see if @holmanb has strong opinions.

Reply:
Honestly that's fair. I did not realize it slowed it down that much. (after some quick testing locally, goes from roughly 18s to 24s with --cov)